### ISSUE TITLE
Uncaught TypeError: Cannot read property 'call' of undefined

### ISSUE THREAD

**Issue by AliSaleem27** [laravel/nova-issues#499]
I have added nova to a fresh instance of Laravel but get the following errors on the nova dashboard.

```
Uncaught TypeError: Cannot read property 'call' of undefined
    at t (manifest.js?id=2d315777967b6001bcf1:1)
    at Object../resources/js/app.js (app.js?id=5a12a2096a117db587e7:47543)
    at t (manifest.js?id=2d315777967b6001bcf1:1)
    at Object.0 (app.js?id=5a12a2096a117db587e7:54517)
    at t (manifest.js?id=2d315777967b6001bcf1:1)
    at window.webpackJsonp (manifest.js?id=2d315777967b6001bcf1:1)
    at app.js?id=5a12a2096a117db587e7:1

t | @ | manifest.js?id=2d31577…:1
  | ./resources/js/app.js | @ | app.js?id=5a12a20…:47543
  | t | @ | manifest.js?id=2d31577…:1
  | 0 | @ | app.js?id=5a12a20…:54517
  | t | @ | manifest.js?id=2d31577…:1
  | window.webpackJsonp | @ | manifest.js?id=2d31577…:1
  | (anonymous) | @ | app.js?id=5a12a20…:1
```

```
Uncaught ReferenceError: CreateNova is not defined
    at nova:130
(anonymous) @ nova:130
```

```
Uncaught ReferenceError: Nova is not defined
    at nova:137
(anonymous) @ nova:137
```

Laravel: 5.5.7
Nova: 1.0.13
npm : 6.4.1
node: 10.10.0
![nova_error](https://user-images.githubusercontent.com/28596110/45266764-2541ef80-b458-11e8-8e99-46f79999152f.png)

**Comment by rivenme** [comment]
Did you find solution? I got same problem. was working ok until I upgraded to 0.13.

**Comment by AliSaleem27** [comment]
I've just tried the same thing with nova version 1.0.12 and it works. So the issue must have been introduced in 1.0.13

**Comment by AliSaleem27** [comment]
Duplicate of #482

**Comment by rivenme** [comment]
fixed by follow  #482

**Comment by Diddyy** [comment]
Fixed with new Release 1.0.14

**Comment by davidhemphill** [comment]
Make sure to run `php artisan nova:publish --force` to get the latest assets after update.
